pkg/db/vulnerability.go

```diff
@@ -1,14 +1,38 @@
 package db
 
+type FixState string
+
+const (
+	IndeterminateFixState FixState = "fixes-indeterminate"
```

Comment:
This reads somewhat odd to me. If it is  an _indeterminate fix state_ (singular) why is the value `"fixes-indeterminate"` (plural) backwards?

Reply:
the `fixes-indeterminate` seems more accurate since there could be more than one fix available. In general this is stating "we can't determine the state of any fix".... I feel that `indeterminate-fix` wouldn't be as accurate (splitting hairs here).

Reply:
I'm having a hard time with `fixes-indeterminate` as well. Can we say `unknown` or something? We're already in the context of the `Fix` block, is that right?

Reply:
What I notice here is that the value and the variable naming are at odds. If we _"can't determine the state of any fix"_ which implies "one or many", then wouldn't it be accurate to say that the state is `"indeterminate"`? 

If this really wants to refer to "fixes" (plural) then it would be easier to read if the constant reflects the plural use there as well.

Reply:
I think unknown will do alright 👍

Reply:
updated